CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# How to Contribute
+
+Contributing to this project entails submitting pull requests (PRs) to this repository. PRs need to be appropriately assigned and labelled.
+
+1. Fork this repository.
+2. Clone your fork of the repository.
+3. When developing, develop and commit in a new, local branch. Leave the protected branches (e.g. develop) alone. They are needed to fetch from upstream.
+   1. If your changes are comprised of multiple commits, use `git rebase -i` to reduce the changes to a single commit. Otherwise, use `git amend` to add files to a commit.
+4. When you are ready to make a PR, fetch from upstream to make sure you have the latest changes from the upstream repository. Add your commit on top of any new commits: `git cherrypick <commit hash>`
```

Comment:
I'd say that you should use `rebase`. Assuming that the remote is `origin` (you may name it differently for fork):
```
git fetch origin
git rebase origin/develop
git checkout my_branch
git rebase develop
```

Cherrypick is useful but not day-to-day, at least in my opinion.

Reply:
Changed instructions to use `git rebase`.